### PR TITLE
Pci config space v2

### PR DESCRIPTION
**Pull request by narasimhan-v**

### CODE REVIEW

**Comment by narasimhan-v** [comment]
[PCI_Info.txt](https://github.com/avocado-framework-tests/avocado-misc-tests/files/986131/PCI_Info.txt)

File: io/pci/pci_info_lsvpd.py [hunk 1]
```diff
@@ -46,68 +46,68 @@ def test(self):
         Test
         '''
         error = []
-        for pci_adres in pci.get_pci_addresses():
-            print "Comparing for PCI Adress : ", pci_adres, "\n\n"
-            vpd_output = pci.get_vpd(pci_adres)
+        for pci_addr in pci.get_pci_addresses():
+            print "Checking for PCI Address : ", pci_addr, "\n\n"
```
Comment (vrbagalkote) [inline io/pci/pci_info_lsvpd.py]: Can this print also be self.log.info()

File: io/pci/pci_info_lsvpd.py [hunk 2]
```diff
@@ -46,68 +46,68 @@ def test(self):
...
+            vpd_output = pci.get_vpd(pci_addr)
             if vpd_output:
-                '''
-                Slot Match
-                '''
+
+                # Slot Match
```
Comment (vrbagalkote) [inline io/pci/pci_info_lsvpd.py]: these empty lines are required??

Comment (narasimhan-v) [inline io/pci/pci_info_lsvpd.py]: Just for good viewing of code :)

**Comment by PraveenPenguin** [comment]
@vrbagalkote thanks for review

**Pull request merged**